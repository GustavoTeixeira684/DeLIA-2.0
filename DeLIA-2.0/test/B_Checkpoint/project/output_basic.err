Any local data was saved: No such file or directory
DeLIA0x603b18ee9970
DeLIA0x603b18eeb830
DeLIA0x603b18eeb890
DeLIA0x603b18eeb8c0
