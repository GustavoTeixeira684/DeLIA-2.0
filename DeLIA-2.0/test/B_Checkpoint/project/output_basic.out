PASSOU AQUI FDP!
============================================
-----	 FAULT TOLERANCE 	-----
 FT_FOLDER 				 = ./data
 CHECKPOINTING_TIME_GLOBAL 		 = -1
 CHECKPOINTING_TIME_LOCAL 		 = -1
 CHECKPOINTING_TIME_DIFF_LOCAL 		 = -1
 CHECKPOINTING_GLOBAL_ITERATION 	 = 1
 TIME_SHARING_LOCAL_DATA_NEIGHBOR 	 = -1
 CHECKPOINTING_LOCAL_RECENT 		 = 1
 WORKLOAD_SCHEDULER 			  = DS
 WORKLOAD_SCHEDULER_LEADER 		 = -1
 TRIGGER_SIGNAL 			 = 0
 TRIGGER_HEARTBEAT_MONITORING:
	 TYPE 				 = 
	 TIME_MAX_WAIT			 = -1
	 SLEEP_THREAD_TIME		 = -1
============================================
ID: 0 Ã‰ servidor com clientes: 0 
0
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 1		-----
============================================
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 2		-----
============================================
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 3		-----
============================================
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 4		-----
============================================
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 5		-----
============================================
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 6		-----
============================================
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 7		-----
============================================
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 8		-----
============================================
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 9		-----
============================================
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 10		-----
============================================
TEST FINALIZED OK
